odoo/defaults/main.yml

```diff
@@ -16,7 +16,7 @@ odoo_config_path: /etc/odoo
 odoo_modules_path: /opt/odoo/modules
 
 odoo_log_path: /var/log/odoo
-odoo_log_level: warning
+odoo_log_level: warn
```

Comment:
Should we start using a more verbose log level? Just to make sure we don't miss relevant log info.